### ISSUE TITLE
SummerBoy.code

### ISSUE THREAD

**Issue by zhpmatrix** [zhpmatrix/pac2017#2]
# reshape X to be [samples, time steps, features]
X = numpy.reshape(dataX, (n_patterns, seq_length, 1))

model = Sequential()
model.add(LSTM(256, input_shape=(X.shape[1], X.shape[2])))
model.add(Dropout(0.2))
model.add(Dense(y.shape[1], activation='softmax'))
model.compile(loss='categorical_crossentropy', optimizer='adam')

model.fit(X, y, nb_epoch=20, batch_size=128, callbacks=callbacks_list)

**Comment by zhpmatrix** [comment]
实验证明：

               将句向量作为LSTM的输入，训练acc比采用Embedding层+词索引序列的acc要低0.001+。
